### DISCUSSION TITLE
Welcome to moneymanagerex Discussions!

### DISCUSSION THREAD

**Discussion by vomikan** [moneymanagerex/moneymanagerex#2731] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by vomikan** [comment]
@whalley 
I've created new set of icons
https://github.com/vomikan/moneymanagerex/blob/master/resources/themes/simple-black/readme.md
Now it may be implemented a style schoosing.

**Reply by whalley** [reply]
Yes, I was just about to start looking at adding the ability to add user selection of theme.... Will hurry this along :-)

**Reply by vomikan** [reply]
@whalley If you don't mind, I'll make a commit.

**Reply by vomikan** [reply]
I suggest to rename "clean-duo" to "default".Another one to "alternate".

![image](https://user-images.githubusercontent.com/6836805/110538719-66084580-8135-11eb-9a69-080225501cc4.png)


This UI remake to use wxChoice instead wxRadioBox. Then add one more wxChoice to choose a style.

**Reply by whalley** [reply]
Go for it.

**Reply by whalley** [reply]
Good with default as name but suggest others have unique names so we can add others, a name of alternate is only useful if we have 2

**Comment by vomikan** [comment]
@whalley I think it's time to release a new release.

There is only one bug left.
https://github.com/moneymanagerex/moneymanagerex/milestone/15

But it's super bug. I does not know how to fix it. Hello to @stef145g

**Reply by whalley** [reply]
Just about to commit a fix

**Comment by tosim91** [comment]
Been using MMEx since it first came out, on Windows, then on Linux. I've stuck with the 1.40 release all this time, since it worked real good for me all along. I've been in VA hospital for awhile, and got home thgis AM. Upon booting up MMEx, I'm finding it is now v1.5.1, and NOT ACCEPTING MY DB. Even though I requested the debug file, it DOES NOT send it. As I've got a consierable DB, I DO NOT wish to redo from scratch. Will somebody PLEASE send me a link to the 1.4.0deb download link, as I'm not able to find one. Thank you.

**Reply by whalley** [reply]
Use the debug file here: www.ipx.co.uk/mmex/downgrade_db_13_7_v4.mmdbg
